src/components/DeveloperModeModal.tsx

```diff
@@ -31,68 +17,68 @@ const ENVIRONMENTS: Environment[] = [
   {
     key: 'app',
     title: 'app',
-    url: SHAPESHIFT_URI,
+    url: process.env.EXPO_PUBLIC_SHAPESHIFT_URI,
   },
   {
     key: 'private',
     title: 'private',
-    url: SHAPESHIFT_PRIVATE_URI,
+    url: process.env.EXPO_PUBLIC_SHAPESHIFT_PRIVATE_URI,
   },
   {
     key: 'release',
     title: 'release',
-    url: RELEASE_URI,
+    url: process.env.EXPO_PUBLIC_RELEASE_URI,
   },
   /**
    * shared development environments
    */
   {
     key: 'dev',
     title: 'develop',
-    url: DEVELOP_URI,
+    url: process.env.EXPO_PUBLIC_DEVELOP_URI,
   },
   {
     key: 'yeet',
     title: 'yeet',
-    url: YEET_URI,
+    url: process.env.EXPO_PUBLIC_YEET_URI,
   },
   /**
    * individual ephemeral environments
    */
   {
     key: 'cafe',
     title: 'cafe',
-    url: CAFE_URI,
+    url: process.env.EXPO_PUBLIC_CAFE_URI,
   },
   {
     key: 'beard',
     title: 'beard',
-    url: BEARD_URI,
+    url: process.env.EXPO_PUBLIC_BEARD_URI,
```

Comment:
We can't destructure this thing, expo don't support it, we need to consume them as I did